base/core/src/main/java/org/openscience/cdk/atomtype/CDKAtomTypeMatcher.java

```diff
@@ -1016,7 +1017,7 @@ private boolean isSingleHeteroAtom(IAtom atom, IAtomContainer container) {
 
     }
 
-    private boolean isRingAtom(IAtom atom, IAtomContainer atomContainer, RingSearch searcher) {
+    private boolean isRingAtom(IAtom atom, IAtomContainer atomContainer, RingSearch searcher) throws NoSuchAtomException {
```

Comment:
Nope!!!! I'm actually about the change the API to try and make NoSuchAtom an unchecked exception but please watch out for breaking the API.